app/controllers/green_flag/admin/features_controller.rb

```diff
@@ -3,7 +3,7 @@ class GreenFlag::Admin::FeaturesController < ApplicationController
   layout 'green_flag/application'
 
   def index
-    @features = GreenFlag::Feature.all
+    @features = GreenFlag::Feature.order(:created_at).all
```

Comment:
We should probably include the rules in this query, since we're using them below.